strings/memories.json

```diff
@@ -411,5 +411,14 @@
 	"supermatter_dusted_moods":[
 		"%PROTAGONIST %MOOD as they faded way.",
 		"%PROTAGONIST %MOOD as they are reduced to atoms."
+	],
+	"nuke_code_names":[
+		"The detonation code for the %PROTAGONIST, %NUKE_CODE" 
+	],
+	"nuke_code_starts":[
+		"%PROTAGONIST starts grinning as they recall the code for detonating a nuclear device, %NUKE_CODE"
+	],
+	"nuke_code_moods":[
+		"%PROTAGONIST %MOOD as they recalled the codes for detonating a nuclear device, %NUKE_CODE"
```

Comment:
i know i said it was fine while starting but actually you should quickly relook through this section and rewrite the moods, starts, and names to not be so... janky!

if someone engraves their memory, it will come out as
```
The detonation code for the John Doe, 4502"

"John Doe starts grinning as they recall the code for detonating a nuclear device, 4502.
they smile slightly as they recalled the codes for detonating a nuclear device, 4502"
```

That's pretty bad! Take a look at the `account` entries, it's really close to what you're doing basically

Reply:
if you wanna go above and byond:tm: can you include multiple lines like the other entries so there's some variety on engraving this?